packages/react-ui-ag/src/Listings/DesktopListing.js

```diff
@@ -47,8 +47,7 @@ export default class DesktopListing extends PureComponent {
           <ListingComponents.Availability /> :
           (
             <div className={theme.UnitLevelAvailabilityAndLastUpdated}>
-              <ListingComponents.UnitLevelAvailability />
```

Comment:
Removed ULA component since its being replaced with `ulaText`

Reply:
Can we remove the component then from `ListingComponents`?

Reply:
On second thought, just leave it. It's not hurting anything.